jib-cli/src/test/java/com/google/cloud/tools/jib/cli/jar/JarModeProcessorTest.java

```diff
@@ -578,4 +578,20 @@ public void testCreateLayersForExplodedNonLayeredSpringBoot()
             AbsoluteUnixPath.get("/app/BOOT-INF/classes/class1.class"),
             AbsoluteUnixPath.get("/app/BOOT-INF/classes/classDirectory/class2.class"));
   }
+
+  @Test
+  public void testCreateLayersForPackagedSpringboot() throws IOException, URISyntaxException {
```

Comment:
```suggestion
  public void testCreateLayersForPackagedSpringBoot() throws IOException, URISyntaxException {
```

And probably `IOException` is unnecessary?